lib/controllers/posts.js

```diff
@@ -0,0 +1,21 @@
+const { Router } = require('express');
+const Post = require('../models/Post');
+const authenticate = require('../middleware/authenticate');
+
+module.exports = Router()
+  .get('/', authenticate, async (req, res, next) => {
+    try {
+      const data = await Post.getAll();
+      res.json(data);
+    } catch (e) {
+      next(e);
+    }
+  })
+  .post('/', async (req, res, next) => {
+    try {
+      const data = await Post.insert(req.body);
```

Comment:
Because we have access to the req.body - and the acceptance criteria mention a limit to 250 characters - this could be a spot to consider adding an if block (since you could access how long a users post is by reading it off of req.body), which returns an error if the post length is over 250 chars